design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,109 @@
+# Implementing Authenticator Status Checks
+The authenticator status check allows operators to check if the authenticator is configured correctly. With this capability, 
```

Comment:
... allows a Conjur operator to get an immediate feedback on the authenticator configuration.
Once the configuration process is complete, the operator sends a RESTful request using the Conjur API that is responded with the status of the given OIDC authenticator.